lib/data_logger/destination/controller.ex

```diff
@@ -13,6 +13,7 @@ defmodule DataLogger.Destination.Controller do
   `destination` when the task finishes.
   It will also react when the task is `:DOWN`.
   """
+  @timeout 1000
```

Comment:
Is this a bit short?

Reply:
Originally it was set at one minute, but after writing the test it became too slow so I toned it down, but obviously it's something that the test made me do.

But thinking of it's usage we have those 2 situations in the system as most common:
1) We get multiple events within milliseconds at that's when we create the SubsetLogged events, etc.
2) We get few events from external sources like installing an asset.

I think the 1 second works nicely in both these cases.

I think I will make it configurable at runtime in a later PR so that it's easy to test and it would have better API if I do it.
Question is whether I want to do it in this PR or the next :thinking: